src/test/java/seedu/address/testutil/McGymmyBuilder.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.testutil;
+
+import seedu.address.model.McGymmy;
+import seedu.address.model.person.Person;
+
+/**
+ * A utility class to help with building McGymmy objects.
+ * Example usage: <br>
+ *     {@code McGymmy ab = new McGymmyBuilder().withPerson("John", "Doe").build();}
+ */
+public class McGymmyBuilder {
+
+    private McGymmy mcGymmy;
+
+    public McGymmyBuilder() {
+        mcGymmy = new McGymmy();
+    }
+
+    public McGymmyBuilder(McGymmy mcGymmy) {
+        this.mcGymmy = mcGymmy;
+    }
+
+    /**
+     * Adds a new {@code Person} to the {@code McGymmy} that we are building.
+     */
+    public McGymmyBuilder withPerson(Person person) {
```

Comment:
I think this should be withFood or something

Also is using `person` here intended?

Reply:
I was just changing the name of the class. I haven't changed any of its implementations.